numpy/core/fromnumeric.py

```diff
@@ -2398,6 +2398,39 @@ def any(a, axis=None, out=None, keepdims=np._NoValue, *, where=np._NoValue):
     >>> np.any([[True, False], [False, False]], where=[[False], [True]])
     False
 
+    >>> a1 = np.array([1, 0, 1], dtype=bool)
+    >>> a1
+    array([ True, False,  True])
+    >>> np.any(a1)
+    True
+
+    >>> a2 = np.array([[1, 0, 0], [1, 0, 0], [0, 0, 1]], dtype=bool)
+    >>> a2
+    array([[ True, False, False],
+    ...    [ True, False, False],
+    ...    [False, False,  True]])
+    >>> np.any(a2, axis=0)
+    array([ True, False,  True])
+    >>> np.any(a2, axis=1)
+    array([ True,  True,  True])
+
+    >>> a3 = np.array([[[1, 0, 0], [1, 0, 0]],
+    ...                 [[0, 0, 1], [0, 0, 1]]], dtype=bool)
+    >>> a3
+    array([[[ True, False, False],
+    ...     [ True, False, False]],
+    ...     [[False, False,  True],
+    ...     [False, False,  True]]])
+    >>> np.any(a3, axis = 0)
+    array([[ True, False,  True],
+    ...    [ True, False,  True]])
+    >>> np.any(a3, axis = 1)
+    array([[ True, False, False],
+    ...    [False, False,  True]])
+    >>> np.any(a3, axis = 2)
+    array([[ True,  True],
+    ...    [ True,  True]])
+    
```

Comment:
I would say you should remove this. I see no added value compared to the example `a2`.